drivers/builtin/src/crypto_oid.h

```diff
@@ -28,31 +28,7 @@
  * the public API, Hide it from scripts/generate_errors.pl, which
  * is in the mbedtls repository, and would complain because it doesn't
  * recognize the header name "crypto_oid.h". */
```

Comment:
This comment is obsolete since the definition of `MBEDTLS_ERR_OID_NOT_FOUND` is no longer visible to `generate_errors.pl`.